warlock.trigger.xml

```diff
@@ -364,7 +364,7 @@ resetFormat()
                     <string>Odpowiadasz</string>
                     <string>przemawia do Ciebie</string>
                     <string>Przemawiasz</string>
-                    <string>@</string>
+                    <string>[</string>

```

Comment:
Nie rozumiem, dlaczego tak?

Reply:
Dobra, rozumiem, ale to nie jest wystarczające, oraz trzeba poprawić też warunek poniżej.